composer.json

```diff
@@ -40,20 +40,20 @@
         "wiki": "https://github.com/Atlantic18/DoctrineExtensions/tree/main/doc"
     },
     "require": {
-        "php": "^7.4 || ^8.0",
-        "behat/transliterator": "^1.2",
-        "doctrine/collections": "^1.2 || ^2.0",
+        "php": "^8.1",
+        "behat/transliterator": "^1.3",
+        "doctrine/collections": "^1.6.5 || ^2.0",
```

Comment:
Just for reference, all the changes to the minimum versions are either to get to a functional set of dependencies on PHP 8.1 (the transliterator package has errors on older versions, `doctrine/common` doesn't support some PHP 8.1 reflection changes until 3.1) or change to match the versions currently being installed on the `prefer-lowest` build.  Considering how old some of these releases are it shouldn't be a big deal, I hope.